src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Education and Work Plan API
-  version: '1.15.4'
+  version: '1.16.0'
```

Comment:
Bump the version of the swagger spec (semver)
Technically this is a breaking change to the API (though it is actually fixing/improving things 😁 ), so this feels like the right version number change